apps/dashboard/src/app/account/billing/checkout/checkout.component.ts

```diff
@@ -390,15 +398,19 @@ export class CheckoutComponent implements OnDestroy, OnInit {
           this.modalService.openModal(ReviewComponent, {
             dismissable: true,
             data: {
-              account: this.accountType,
+              account: this.accountType!,
               customer: this.customer,
               token: this.token,
-              prices: prices[this.accountType],
+              prices: prices[this.accountType!],
```

Comment:
not a big deal, but ESlint is unhappy with `!` saying 
> ESLint: Forbidden non-null assertion.(@typescript-eslint/no-non-null-assertion)

I usually try to avoid them as mush as possible (I don't like having code highlighted by ESLint)

Reply:
Fixed